### PR TITLE
Completed the weather app template code

### PR DESCRIPTION
**Pull request by bhukya-sainaveen**
# The weather app template
### Tech Stack - React, Bootstrap, API Calls
The weather app uses [OpenWeatherMap](https://openweathermap.org/api) API key to describe the current weather.
The API key can be set in the config.js file.

### CODE REVIEW

File: React/weather-app/.gitignore
Comment (TheGuyDangerous) [inline React/weather-app/.gitignore:1]: Make sure to not include ```.gitignore``` files while creating a PR

**Review by TheGuyDangerous — CHANGES_REQUESTED** [review]
Please include your ```weather-app``` project in the ```index.html``` file and make sure it works properly, when opened on a local host server, without any background server needing to be opened.

**Review by bhukya-sainaveen — COMMENTED** [review]
Modified the index.html to support the weather app and also tested the website using the local server to see whether the code was working without any background server needing to be opened.

**Review by TheGuyDangerous — APPROVED** [review]
All  good ✅

**Pull request merged**